### PR TITLE
chore: remove name and picture and choose your adventure experiment

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Removes the Choose your own adventure (CYA) experiment code and makes the screen permanent. Also removes the now unused name and picture onboarding screen and FormInput component.

### Test plan

- Tested locally on iOS.
- E2E tests updated.
- Unit tests updated.

### Related issues

- Fixes act-1114

### Backwards compatibility

Yes

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/onboarding/registration/RegulatoryTerms.test.tsx [hunk 1]
```diff
@@ -30,10 +30,10 @@ describe('RegulatoryTermsScreen', () => {
       fireEvent.press(wrapper.getByTestId('AcceptTermsButton'))
       expect(acceptTerms).toHaveBeenCalled()
     })
-    it('navigates to NameAndPicture', () => {
+    it('navigates to ImportSelect', () => {
       const store = createMockStore({})
       const acceptTerms = jest.fn()
-      jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.NameAndPicture)
+      jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.ImportSelect)
```
Comment (satish-ravi) [inline src/onboarding/registration/RegulatoryTerms.test.tsx:36]: shouldn't this be PincodeSelect?

File: src/onboarding/registration/RegulatoryTerms.tsx [hunk 2]
```diff
@@ -83,7 +76,7 @@ export class RegulatoryTerms extends React.Component<Props> {
 
     return (
       <SafeAreaView style={styles.container}>
-        <DevSkipButton nextScreen={Screens.NameAndPicture} />
+        <DevSkipButton nextScreen={Screens.ImportSelect} />
```
Comment (satish-ravi) [inline src/onboarding/registration/RegulatoryTerms.tsx:79]: shouldn't this be PincodeSet?

File: src/onboarding/welcome/Welcome.test.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ describe('Welcome', () => {
         acceptedTerms: true,
       },
     })
-    jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.NameAndPicture)
+    jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.ImportSelect)
```
Comment (satish-ravi) [inline src/onboarding/welcome/Welcome.test.tsx:61]: same as above, should this be PincodeSet?

File: src/onboarding/steps.ts [hunk 4]
```diff
@@ -317,7 +277,7 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
             // skipping phone verification. Every other case navigates to CYA at
             // the end of onboarding, including when skipping verification on
             // the import flow. TODO(ACT-1114): make this consistent with other cases
```
Comment (satish-ravi) [inline src/onboarding/steps.ts:279]: can we remove the todo?

**Review by satish-ravi — COMMENTED** [review]
Nice cleanup!

File: src/onboarding/registration/RegulatoryTerms.test.tsx (hunk 1 above)
Comment (MuckT) [reply src/onboarding/registration/RegulatoryTerms.test.tsx]: Fixed in 8767c90!

File: src/onboarding/welcome/Welcome.test.tsx (hunk 3 above)
Comment (MuckT) [reply src/onboarding/welcome/Welcome.test.tsx]: Fixed in 8767c90!

File: src/onboarding/steps.ts (hunk 4 above)
Comment (MuckT) [reply src/onboarding/steps.ts]: Removed in 20c7513!

File: src/onboarding/registration/RegulatoryTerms.tsx (hunk 2 above)
Comment (MuckT) [reply src/onboarding/registration/RegulatoryTerms.tsx]: Fixed in 512a166!

**Review by satish-ravi — APPROVED** [review]

**Pull request merged**